<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QvisNoiseReductionDialog</class>
 <widget class="QDialog" name="QvisNoiseReductionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noise Reduction</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/Src/FluoRender/UI/icons/Tools/NoiseReduction.png</selectedon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <item>
    <layout class="QGridLayout" name="ValuesLayout">
     <item row="0" column="2">
      <widget class="QSpinBox" name="ThresholdSpinBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="ThresholdSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ThresholdLabel">
       <property name="text">
        <string>Threshold</string>
       </property>
       <property name="buddy">
        <cstring>ThresholdSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="VoxelSizeLabel">
       <property name="text">
        <string>Voxel Size</string>
       </property>
       <property name="buddy">
        <cstring>VoxelSizeSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="VoxelSizeSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="VoxelSizeSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="SelectionLayout">
     <item>
      <widget class="QCheckBox" name="SlectedRegionOnlyCheckBox">
       <property name="text">
        <string>Selected region only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="EnhanceSelectionCheckBox">
       <property name="text">
        <string>Enhance Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="SelectionSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ButtonLayout">
     <item>
      <widget class="QPushButton" name="EraseButton">
       <property name="text">
        <string>Erase</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PreviewButton">
       <property name="text">
        <string>Preview</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="ButtonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ThresholdSlider</tabstop>
  <tabstop>ThresholdSpinBox</tabstop>
  <tabstop>VoxelSizeSlider</tabstop>
  <tabstop>VoxelSizeSpinBox</tabstop>
  <tabstop>SlectedRegionOnlyCheckBox</tabstop>
  <tabstop>EnhanceSelectionCheckBox</tabstop>
  <tabstop>EraseButton</tabstop>
  <tabstop>PreviewButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ThresholdSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisNoiseReductionDialog</receiver>
   <slot>ThresholdSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ThresholdSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisNoiseReductionDialog</receiver>
   <slot>ThresholdValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VoxelSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisNoiseReductionDialog</receiver>
   <slot>VoxelSizeSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VoxelSizeSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QvisNoiseReductionDialog</receiver>
   <slot>VoxelSizeValueChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SlectedRegionOnlyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisNoiseReductionDialog</receiver>
   <slot>SlectedRegionOnlyToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EnhanceSelectionCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisNoiseReductionDialog</receiver>
   <slot>EnhanceSelectionToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EraseButton</sender>
   <signal>clicked()</signal>
   <receiver>QvisNoiseReductionDialog</receiver>
   <slot>EraseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PreviewButton</sender>
   <signal>clicked()</signal>
   <receiver>QvisNoiseReductionDialog</receiver>
   <slot>PreviewButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ThresholdSliderChanged(int)</slot>
  <slot>ThresholdValueChanged(int)</slot>
  <slot>VoxelSizeSliderChanged(int)</slot>
  <slot>VoxelSizeValueChanged(double)</slot>
  <slot>SlectedRegionOnlyToggled(bool)</slot>
  <slot>EnhanceSelectionToggled(bool)</slot>
  <slot>EraseButtonClicked()</slot>
  <slot>PreviewButtonClicked()</slot>
 </slots>
</ui>
